include/exadg/time_integration/time_int_gen_alpha_base.h

```diff
@@ -53,17 +53,26 @@ class TimeIntGenAlphaBase : public TimeIntBase
   set_current_time_step_size(double const & time_step_size) final;
 
   double
-  get_scaling_factor_mass() const;
+  get_scaling_factor_mass_from_acceleration() const;
+
+  double
+  get_scaling_factor_mass_from_velocity() const;
```

Comment:
I do not see why to add a new function? Adding the two scaling factors appears twice in this PR, so this is duplicated code which would not occur when adding the two factors inside the pre-existing function `get_scaling_factor_mass()`.